src/Components/Connexion.jsx

```diff
@@ -0,0 +1,32 @@
+import logoFavart from "../Assets/logoFavart.jpg";
+
+import {
+  Background,
+  ContainerInscription,
+  ContainerForm,
+  Logo,
+  ContainerLogo,
+  Slogan,
+  Span,
+  TitleForm,
+  H2,
+} from "../Styled-components/ConnexionStyle";
+
+export default function connexion() {
```

Comment:
This is a React component: rename it to "Connection" (first capital letter + English name)